### PR TITLE
Updating pom.xml to consume iotdevicesdk 1.2.10 version from maven ce…

### PR DESCRIPTION
**Pull request by abanthiy**
…ntral

**Issue #, if available:**
Fast follow on consuming iotdevicesdk from maven central

**Description of changes:**
Updating iotdevicesdk dependency to version 1.2.10

**Why is this change necessary:**

**How was this change tested:**

**Any additional information or context required to review the change:**

**Checklist:**
 - [ ] Updated the README if applicable
 - [ ] Updated or added new unit tests
 - [ ] Updated or added new integration tests
 - [ ] Updated or added new end-to-end tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by MikeDombo — DISMISSED** [review]
Remove the todo, also switch the AWS SDK to consume?

**Comment by fengwa-aws** [comment]
Can you use https://www.conventionalcommits.org/en/v1.0.0/ to write the commit message?

File: pom.xml [hunk 1]
```diff
@@ -192,6 +191,16 @@
...
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+            <id>central</id>
+            <url>https://repo1.maven.org/maven2</url>
+        </pluginRepository>
         <pluginRepository>
```
Comment (MikeDombo) [inline pom.xml:204]: I'm pretty sure that this second repo can be removed now.

File: pom.xml [hunk 2]
```diff
@@ -112,9 +112,9 @@
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
         <dependency>
-            <groupId>com.amazonaws.services</groupId>
+            <groupId>software.amazon.awssdk</groupId>
             <artifactId>greengrassv2</artifactId>
-            <version>2.0.0-SNAPSHOT</version>
+            <version>2.15.47</version>
```
Comment (MikeDombo) [inline pom.xml:117]: remove the version here. It comes from the version on L31

**Review by MikeDombo — APPROVED** [review]

**Review by jbutler — APPROVED** [review]

**Pull request merged**